CriteoPublisherSdk/Tests/IntegrationTests/CR_WebApiIntegrationTests.m

```diff
@@ -52,17 +52,13 @@ @implementation CR_WebApiIntegrationTests
 - (void)setUp {
   NSString *userDefaultsId = NSStringFromClass(self.class);
   [[NSUserDefaults new] removePersistentDomainForName:userDefaultsId];
```

Comment:
This line seems to clean the state of the user default created with the userDefaultsId.
After your changes, the user defaults are created by the dependency provider, and is not using this userDefaultsId anymore.
I think that you should either remove this cleaning if it is not needed anymore.
Or, this cleaning is still required, then a solution should be found to inject the proper userDefaults.

Reply:
Same concern.

Reply:
Removed as it is useless now